manifests/server.pp

```diff
@@ -297,6 +298,23 @@
     }
   }
 
+  if $automatic_dlo_cleanup {
+    if $facts['systemd'] {
+      # deploy a systemd timer + service to cleanup old reports
+      # https://puppet.com/docs/puppetdb/5.2/maintain_and_tune.html#clean-up-the-dead-letter-office
+      systemd::unit_file{'puppetdb-cleanup.service':
+        content => epp("${module_name}/puppetdb-DLO-cleanup.service.epp"),
+      }
+      -> systemd::unit_file{'puppetdb-cleanup.timer':
+        content => epp("${module_name}/puppetdb-DLO-cleanup.timer.epp"),
+      }
+      -> service{'puppetdb-cleanup.timer':
```

Comment:
This needs to be a `~>` so that the service will be restarted if the `puppetdb-cleanup.timer` file changes.

Reply:
After months of work, this feature is finally released. It wasn't available when when I created this PR. updated the code.